src/dotnet/ShellShimMaker/LinuxEnvironmentPath.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.ShellShimMaker
+{
+    internal class LinuxEnvironmentPath : IEnvironmentPath
+    {
+        private readonly IFile _fileSystem;
+        private readonly IEnvironmentProvider _environmentProvider;
+        private readonly IReporter _reporter;
+        private const string PathName = "PATH";
+        private readonly string _packageExecutablePath;
+        private const string ProfiledDotnetCliToolsPath = @"/etc/profile.d/dotnet-cli-tools-bin-path.sh";
+
+        internal LinuxEnvironmentPath(
+            string packageExecutablePath,
+            IReporter reporter,
+            IEnvironmentProvider environmentProvider,
+            IFile fileSystem)
+        {
+            _fileSystem = fileSystem ?? throw new ArgumentNullException(nameof(fileSystem));
+            _environmentProvider
+                = environmentProvider ?? throw new ArgumentNullException(nameof(environmentProvider));
+            _reporter
+                = reporter ?? throw new ArgumentNullException(nameof(reporter));
+            _packageExecutablePath
+                = packageExecutablePath ?? throw new ArgumentNullException(nameof(packageExecutablePath));
+        }
+
+        public void AddPackageExecutablePathToUserPath()
+        {
+            if (PackageExecutablePathExists()) return;
```

Comment:
Please, put curly braces on the statement.

Reply:
Sorry, that code was before yesterdays code review, I will change it

Reply:
fixed